src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -642,72 +642,6 @@ public async Task HandleRequestAsync_CSharpProjection_TransitionTriggerCharacter
             Assert.Equal(expectedItem.InsertText, item.InsertText);
         }
 
-        [Fact]
-        public async Task HandleRequestAsync_CSharpProjection_ReturnOnlyPropertiesIfInsideInitializer()
```

Comment:
This test was there to check, if post processing adds snippets when requesting completions inside initializer context. Now this logic is fully transfered to LSP, so I see no need for this test anymore